### PR TITLE
Transformations docs

### PR DESCRIPTION
**Pull request by svix-dylan**
Adds documentation for our new Transformations feature.

### CODE REVIEW

File: docs/transformations.mdx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+Transformations are a powerful new Svix feature that allow the modification of certain webhook properties in-flight. When you enable Transformations, your customers can write Javascript code on their endpoints that can change a webhook's HTTP method, target URL, and body payload.
+
+## Enabling Transformations
+
+Transformations can be enabled at the environment-level. When you enable Transformations for an environment, your customers will be able to use Transformations on their endpoints.
+
+To enable Transformations, navigate to the Settings page in the dashboard. Under the Environment Settings section, click General Settings, and then toggle the Enable Transformations switch. Your customers will need to reenter the Consumer App Portal for the change to take effect.
+
+![Management UI screenshot](./img/enable-transformations.png)
```
[the remark this answers is not in the document]
Comment (svix-dylan) [reply docs/transformations.mdx:13]: AH it's copied. Updating

File: docs/transformations.mdx [hunk 2]
```diff
@@ -0,0 +1,63 @@
+---
+title: Transformations
+---
+
+Transformations are a powerful new Svix feature that allow the modification of certain webhook properties in-flight. When you enable Transformations, your customers can write Javascript code on their endpoints that can change a webhook's HTTP method, target URL, and body payload.
+
```
Comment (tasn) [inline docs/transformations.mdx:6]: Add information about it being beta + pricing just like in the screenshot.

File: docs/transformations.mdx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+To enable Transformations, navigate to the Settings page in the dashboard. Under the Environment Settings section, click General Settings, and then toggle the Enable Transformations switch. Your customers will need to reenter the Consumer App Portal for the change to take effect.
+
+![Environment Settings page with the Enable Transformation toggle](./img/enable-transformations.png)
+
+## Using Transformations
+
+Once enabled for an environment, you customers can begin using Transformations by logging into the Consumer Portal, clicking on an endpoint, clicking into the Advanced tab, and scrolling down to the Transformations card:
+
+![The Transformations card on an endpoint](./img/transformations-card.png)
```
Comment (tasn) [inline docs/transformations.mdx:19]: Can you retake these screenshots without the beta tag showing? Just so they are evergreen.

File: docs/transformations.mdx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+![Editing an endpoint's Transformation](./img/edit-transformation.png)
+
+### How to write a Transformation
+
+Svix expects a Transformation to declare a function named `handler`. Svix will pass a `WebhookObject` to this function as its only argument, and expects it to always return a `WebhookObject`.
+
+Here is what a `WebhookObject` looks like:
+
+```json
+{
```
Comment (tasn) [inline docs/transformations.mdx:34]: It's confusing that it shows the payload example, show it with the context of a handler. I know you have the example section below, but still having the json example here is just confusing. I guess actually just having this example is confusing. Just document the fields (like you do in the comment of the default transformation), and then just show an example below, like you have.

**Review by tasn — APPROVED** [review]

**Pull request merged**